website_event_snippet_calendar/controllers/main.py

```diff
@@ -54,12 +55,23 @@ def events_for_day(self, day=None, limit=None):
         :param limit int:
             How many results to return.
         """
+
+        searches = parse_qsl(searches[1:])
```

Comment:
This is error-prone. If you do `None[1:]` you'll get an error.

BTW I think it would be better if the JS client calls this argument directly with an array of results. Passing in a query string as a JSON argument doesn't seem very ergonomic.